__tests__/index.test.ts

```diff
@@ -13,6 +16,13 @@ const config = {
   fetch
 }
 
+function uint8ArrayFromHex(text: string): Uint8Array {
```

Comment:
This helper was nice becuase it makes it easier to copy/paste the hex output from mysql CLI for it's rendering of binary data. Figured I'd like to have a source of truth to copy/paste from rather than hand writing each byte in a Uint8Array.